src/csharp/Grpc.Examples/MathGrpc.cs

```diff
@@ -133,13 +133,13 @@ public AsyncServerStreamingCall<Num> Fib(FibArgs request, CancellationToken toke
         // server-side interface
         public interface IMathService
         {
-            Task<DivReply> Div(DivArgs request);
+            Task<DivReply> Div(ServerCallContext context, DivArgs request);
```

Comment:
What does the Java API look like in terms of this? We might want to consider the other .NET context APIs to make a ServerCallContext implicit and flow with tasks...